src/main/java/net/greatstart/model/User.java

```diff
@@ -49,10 +49,10 @@
     @Embedded
     private Contact contact;
 
-    @OneToMany(mappedBy = "inv", fetch = FetchType.EAGER)
+    @OneToMany(mappedBy = "investor", fetch = FetchType.LAZY)

```

Comment:
this caused me problems when getting User from DB in Forgot Password branch. Please, leave EAGER here and afterwards